### ISSUE TITLE
Sass parser isn't compatible with `@use`

### ISSUE THREAD

**Issue by algomaster99** [stylelint/stylelint#4711]
> Clearly describe the bug

Getting unnecessary `Property should have one space between the prop and the value` when I ran `stylelint` over a `.sass` file.

> Which rule, if any, is the bug related to?

`CssSyntaxError`

> What code is needed to reproduce the bug?

```sass
.cover-button
  display: flex
  justify-content: center
  margin-top: 2.5em
  width: 100%
```

> What stylelint configuration is needed to reproduce the bug?

```json
{
    "plugins": [
      "stylelint-scss"
    ],
    "extends": "stylelint-config-sass-guidelines",
    "rules": {
      "block-opening-brace-space-before": "never",
      "color-hex-length": "long",
      "declaration-block-trailing-semicolon": "never",
      "indentation": 2,
      "max-nesting-depth": 2,
      "scss/dollar-variable-pattern": "[a-zA-Z0-9]+",
      "selector-combinator-space-after": "always",
      "selector-combinator-space-before": "always",
      "selector-list-comma-newline-after": "never-multi-line"
    }
  }
```

> Which version of stylelint are you using?

`13.3.0`

> How are you running stylelint: CLI, PostCSS plugin, Node.js API?

"CLI with `stylelint **/*.{sass,scss} --fix`"

> Does the bug relate to non-standard syntax (e.g. SCSS, Less etc.)?

Yes, it is related to SASS.

> What did you expect to happen?

No errors to be flagged.

> What actually happened (e.g. what warnings or errors did you get)?

`Please check validity of the block starting from line`

```shell
src/components/cover/index.sass
 68:1  ✖  Please check validity of the block starting from line #68   CssSyntaxError
```

Further reference: https://stackoverflow.com/questions/61293176/sugarss-throwing-unecessary-csssyntaxerror/61305305#61305305

**Comment by jeddy3** [comment]
@algomaster99 Thanks for the report and for using the template.

I can reproduce this [using the demo](https://stylelint.io/demo#N4Igxg9gJgpiBcIACBXAzjABAHRAOwEMBbGNABwLBl0wLVoBsBLO7PNjBmMAFwgCc2mTGX4Qy8RizQA6ACQA3AoLwgANCDQBPPDwIAPBJrpp14CHgBmTAOZHgQnCH4ouaXJOABfNl5BegA) with:

```sass
@use "namespace" as alias

selector
  prop: alias.$var
```

The [postcss-sass parser](https://www.npmjs.com/package/postcss-sass) that stylelint uses [to support Sass](https://stylelint.io/about/syntaxes) doesn't appear to understand the [new modules syntax](https://sass-lang.com/documentation/at-rules/use) yet. Incidentally, the [SCSS parser](https://www.npmjs.com/package/postcss-scss) does. The Sass parser uses the Gonzales-PE parser under the hood and [its playground](https://tonyganch.io/gonzales-pe/) can't parse the example above.

Please consider [contributing to the parser](https://stylelint.io/developer-guide/syntaxes) if you have time. You can raise an issue in their [tracker](https://github.com/AleshaOleg/postcss-sass/issues).

As an aside, the Sass syntax isn't as popular as the SCSS one. So support for it in tooling can lag behind that of SCSS.

**Comment by algomaster99** [comment]
@jeddy3 Thanks for replying!
I have opened the ticket for this issue - https://github.com/AleshaOleg/postcss-sass/issues/145. I will try to contribute once this issue is accepted there as well :)

**Comment by JJ810** [comment]
@algomaster99, I am getting same issue in using @import. Is this related to this issue? And Is there any way I can solve this problem?

**Comment by algomaster99** [comment]
@algomaster99 `@import` should work in my opinion. This issue is raised because the parser doesn't support new module syntax. @import is an older syntax.

**Comment by jeddy3** [comment]
The [`syntax` option was removed](https://github.com/stylelint/stylelint/issues/5289) for the [14.0.0 release](https://github.com/stylelint/stylelint/issues/5205). (See the [migration guide](https://github.com/stylelint/stylelint/blob/main/docs/migration-guide/to-14.md#syntax-option-and-automatic-inferral-of-syntax).)

Closing in favour of the upstream issue as the syntax is no longer bundled by Stylelint.
